propulate/propulator.py

```diff
@@ -276,6 +311,11 @@ def _evaluate_individual(self) -> None:
             f"Island {self.island_idx} Worker {self.island_comm.rank} Generation {self.generation}: BREEDING\n"
             f"Bred and evaluated individual {ind}."
         )
+
+        if self.surrogate is not None:
+            # add Surrogate model data to individual for synchronization
+            ind[SURROGATE_KEY] = self.surrogate.data()
```

Comment:
The main advantage of this approach is that a few bytes less are sent every time `ind` is transmitted when not using surrogate models.

Technically there could be a key conflict, but I think it's highly unlikely that a user names a limit "_s" (which could also be changed to something longer - but then again, more bytes send).
Otherwise, using a class attribute would be functionally equivalent.